Source/Fuse.Controls.Native/SystemScreenConfigBase.uno

```diff
@@ -0,0 +1,132 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Fuse;
+
+namespace Fuse.Controls
+{
+	/**
+		TODO write docs please
+	*/
+	public class SystemScreenConfigBase : Behavior
+	{
+		public enum Visibility
+	    {
+	        None,
+	        Minimal,
+	        Full,
+	    }
+
+	    private static List<SystemScreenConfigBase> rootedConfigs = new List<SystemScreenConfigBase>();
```

Comment:
You can use a `MiniList` here for efficiency. I assume 0 and 1 are the most common number of configs used.